docs/README-EDITING.md

```diff
@@ -14,13 +14,13 @@ When adding new documentation files, check the documentation [directory structur
 
 ### Frontmatter
 
-Each Markdown file starts with the following frontmatter tags:
+Each Markdown file can start with the following frontmatter tags:
 
 | Tag            | Meaning                                                    | Required |
-|----------------|------------------------------------------------------------|----------|
-| `title`        | The page's header.                                         | Yes      |
-| `permalink`    | The page's **unique** URL.                                 | Yes      |
-| `canonicalUrl` | A canonical URL of the page's latest version.              | Yes      |
+|----------------|------------------------------------------------------------|:--------:|
+| `title`        | The page's header. If you don't add it,<br>VuePress generates it from the page's parent's title. | Yes      |
+| `permalink`    | The page's **unique** URL. If you don't add it,<br>VuePress generates it from the Markdown file name. | Yes      |
```

Comment:
We mark these fields as required even if we will generate pages without them using default options. I think we should consider using "Default value" instead of "Required". WDYT?

Reply:
@swistach - sure, we can do it like this then: d1711bd